google-cloud-contrib/google-cloud-nio/src/main/java/com/google/cloud/storage/contrib/nio/CloudStorageFileSystemProvider.java

```diff
@@ -572,7 +572,7 @@ public void checkAccess(Path path, AccessMode... modes) throws IOException {
     // null size indicate a file that we haven't closed yet, so GCS treats it as not there yet.
     if (null == blobInfo || blobInfo.getSize() == null) {
       throw new NoSuchFileException(
-          cloudPath.getBlobId().getBucket() + "/" + cloudPath.getBlobId().getName());
+          "gs://" + cloudPath.getBlobId().getBucket() + "/" + cloudPath.getBlobId().getName());
```

Comment:
String.format() is locale-specific, I prefer not using String.format() when it is not purely about human readable messages without technical details (like path) in it. This is inspired by this issue: https://github.com/GoogleCloudPlatform/google-cloud-java/issues/1957

On different locales the outputs of string.format may be unpredictable.